File: evidence/generated-systems/golang/aerospace-on-golang/aerospace-on-golang/internal/model/AerospaceManufacturer.go
package model

import (
    "gorm.io/gorm"
)

//==============================================================
// AerospaceManufacturer Declaration
//==============================================================
type AerospaceManufacturer struct {
    gorm.Model
     Name                                    string
    LegalName                                    string
    HeadquartersCountry                                    string
    Website                                    string
     Programs           []AircraftProgram `gorm:"foreignKey:ProgramsFromAerospaceManufacturerId"`
     Plants           []Plant `gorm:"foreignKey:PlantsFromAerospaceManufacturerId"`
     Suppliers           []Supplier `gorm:"foreignKey:SuppliersFromAerospaceManufacturerId"`
     ProductionCertificates           []ProductionCertificate `gorm:"foreignKey:ProductionCertificatesFromAerospaceManufacturerId"`

// parent associations as their child

}

File: evidence/generated-systems/rubyonrails/aerospace-on-rails/aerospaceonrails/app/models/aerospacemanufacturer.rb
class AerospaceManufacturer < ApplicationRecord


  has_many :Programs, class_name: 'AircraftProgram'
  has_many :Plants, class_name: 'Plant'
  has_many :Suppliers, class_name: 'Supplier'
  has_many :ProductionCertificates, class_name: 'ProductionCertificate'

end
